common/shared.c

```diff
@@ -251,3 +251,90 @@ void show_perf_mr(int tsize, int iters, struct timespec *start,
 	printf("usec/xfer: %f", ((float)elapsed / iters / xfers_per_iter));
 	printf(" }\n");
 }
+
+void ft_csusage(char *name, char *desc)
+{
+	fprintf(stderr, "Usage:\n");
+	fprintf(stderr, "  %s [OPTIONS]\t\tstart ping server\n", name);
+	fprintf(stderr, "  %s [OPTIONS] <host>\tpong given host\n", name);
+
+	fprintf(stderr, "\nOptions:\n");
+	fprintf(stderr, "  -n\tdomain name\n");
+	fprintf(stderr, "  -p\tport number\n");
+	fprintf(stderr, "  -f\tspecific provider\n");
+	fprintf(stderr, "  -s\tsource address\n");
+	fprintf(stderr, "  -I\tnumber of iterations\n");
+	fprintf(stderr, "  -S\tspecific transfer size or 'all'\n");
+	fprintf(stderr, "  -m\tmachine readable output\n");
+	fprintf(stderr, "  -i\tprint hints structure and exit\n");
+	fprintf(stderr, "  -v\tdisplay versions and exit\n");
+	fprintf(stderr, "  -h\tdisplay this help output\n");
```

Comment:
Btw, I realize this is a copy-paste of the existing call.  It's just convenient to fixup while it's being moved.